_pages/docs/techref/ab-file-organization.md

```diff
@@ -754,8 +754,7 @@ permalink:   /docs/techref/file-organization/
 
 <h3>B.3.8 - Maximum sizes</h3>
 
-<p>The max ProDOS volume size is 65,535 blocks, or 32,767.5 KB.</p>
-<p>The maximum size of a volume is 32MB: if pointers are 16 bits and sectors are 512 bytes, the math gives 256 * 256 * 512 = 32MB.</p>
+<p>The maximum size of a volume is 32MB-512 bytes: if pointers are 16 bits and sectors are 512 bytes, the math gives 256 * 256 * 512 = 32MB. Because the number of blocks needs to fit in 2 bytes (total_blocks in Figure B-3), only 65,535 are usable, or 32,767.5 KB.</p>
```

Comment:
The maximum number of addressable blocks is still a full 32 MB. Block $0000 stores the boot loader so only blocks $0001 through $FFFF are available for the volume. ProDOS can access block $0000 to read and write it but it is not part of the volume.